pkg/ocm/ocm.go

```diff
@@ -93,6 +96,18 @@ func ZeroFields(obj runtime.Object) runtime.Object {
 	return zeroed
 }
 
+func GetOCMClientset(kubeconfig *rest.Config) (*clientset.Clientset, error) {
+	clientset, err := clientset.NewForConfig(kubeconfig)
+	if err != nil {
+		return nil, err
+	}
+	return clientset, nil
+}
+
+func GetOCMInformerFactory(clientset clientset.Interface) informers.SharedInformerFactory {
+	return informers.NewSharedInformerFactory(clientset, 0*time.Second)
+}
```

Comment:
same comment as the GetOCMClientset.. no need for this helper function.
just use directly in the controller file.
plus I would extract the 0 value to const. not very critical but I don't like to see hardcoded values in code:
see example here: https://github.com/kubestellar/kubestellar/blob/main/cmd/transport/base-controller.go#L49

Reply:
I would rather keep it in ocm to avoid having to add an extra import into the binding package.  I accepted the adding of the const.

Reply:
that's fine.

btw, putting here a comment just fyi (you don't need to do anything now, just to be aware of this point):
this code has to go through refactoring - 
all clientsets anf informer factory vars should be created in cmd file and not in the controller. 
controller should receive clientsets.interface args instead of restConfig args.
so anyway references to ocm will be in the cmd eventually and not here.

Reply:
Makes sense. That would require changes in the kubeflex workflow.

Reply:
yup. there should be a clear separation between kubeflex and KubeStellar,
which is not the case atm.
there is an open issue for this, see #1533 for more details.